taichi/transforms/type_check.cpp

```diff
@@ -323,8 +327,17 @@ class TypeCheck : public IRVisitor {
     stmt->ret_type = VectorType(1, DataType::i32);
   }
 
+  void visit(GetRootStmt *stmt) {
+    stmt->ret_type = VectorType(1, DataType::gen, true);
+  }
+
+  void visit(SNodeLookupStmt *stmt) {
+    stmt->ret_type = VectorType(1, DataType::gen, true);
```

Comment:
Are we begining to make var Vector(1)?

Reply:
No need to modify anything. This is just a backward-compatible and temporary enhancement to the type system.